source/_components/emulated_roku.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
+title: "Emulated Roku"
+description: "Instructions on how to set up Emulated Roku within Home Assistant."
+date: 2018-10-03 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Hub
+ha_release: 0.80.0
+ha_iot_class: "Local Push"
+ha_qa_scale: internal
+---
+
+This component integrates an emulated Roku API into Home Assistant,  
+so remotes such as Harmony and Android apps can connect to it through WiFi as it were a Roku player.  
+Home Assistant will see key presses and app launches as Events, which you can use as triggers for automations.  
+Multiple Roku servers may be started if you run out of buttons by specifying multiple server entries.  
+When using multiple servers, you can filter the roku USN which is specified via the `name` attribute.
+
+<p class='note'>  
+Windows is not supported because Home Assistant uses `ProactorEventLoop` which does not support UDP sockets.
+</p>  
+
+## {% linkable_title Configuration %}
+
+configuration.yaml
+```yaml
+# Example configuration.yaml entry
+emulated_roku:
+  servers:
+    - name: hass roku
+      listen_port: 8060
+      host_ip: 192.168.1.150
+      advertise_ip: 10.0.0.10
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.